### PR TITLE
Backport 1.4 - Fix metrics deployment in 1.4/3.4 

### PR DESCRIPTION
**Pull request by sdodson**
Backports #2803

### CODE REVIEW

**Comment by sdodson** [comment]
aos-ci-test

File: roles/openshift_metrics/tasks/main.yaml [hunk 1]
```diff
@@ -36,10 +36,8 @@
     metrics_persistence: "{{ openshift.hosted.metrics.storage_kind | default(none) is not none }}"
     metrics_dynamic_vol: "{{ openshift.hosted.metrics.storage_kind | default(none) == 'dynamic' }}"
     metrics_template_dir: "{{ openshift.common.config_base if openshift.common.is_containerized | bool else '/usr/share/openshift' }}/examples/infrastructure-templates/{{ 'origin' if deployment_type == 'origin' else 'enterprise' }}"
-    cassandra_nodes: "{{ ',CASSANDRA_NODES=' ~ openshift.hosted.metrics.cassandra_nodes if 'cassandra' in openshift.hosted.metrics else '' }}"
-    cassandra_pv_size: "{{ ',CASSANDRA_PV_SIZE=' ~ openshift.hosted.metrics.storage_volume_size if openshift.hosted.metrics.storage_volume_size | default(none) is not none else '' }}"
```
Comment (detiber) [inline roles/openshift_metrics/tasks/main.yaml]: Do we not expose these values? seems like we should.

Comment (sdodson) [inline roles/openshift_metrics/tasks/main.yaml]: They were unused, but yeah we definitely provide them as configuration options.

Comment (sdodson) [inline roles/openshift_metrics/tasks/main.yaml]: ...should provide them as configuration options.

Comment (sdodson) [inline roles/openshift_metrics/tasks/main.yaml]: Are you ok merging this as is? This is a clean cherry-pick from master.

**Pull request merged**